### PR TITLE
integrate debounce

### PR DESCRIPTION
**Pull request by rooneyt**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Store, MutationPayload } from "vuex";
 import merge from "deepmerge";
 import * as shvl from "shvl";
+import _ from "lodash";
```
Comment (gacord23) [inline src/index.ts:4]: you'll need to npm intall lodash in order to use this

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,9 @@
 # when working with contributors
 /package-lock.json
 /yarn.lock
+.idea/codeStyles/codeStyleConfig.xml
```
Comment (gacord23) [inline .gitignore]: make .idea removal more comprehensive.  refer to one of our other repos.

**Review by gacord23 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (rooneyt) [reply src/index.ts:4]: oops

File: .gitignore (hunk 2 above)
Comment (rooneyt) [reply .gitignore]: yeah got it.

**Review by gacord23 — APPROVED** [review]

**Pull request merged**